src/OpenTelemetry.Api/Baggage.cs

```diff
@@ -46,21 +46,34 @@ internal Baggage(Dictionary<string, string> baggage)
         /// <summary>
         /// Gets or sets the current <see cref="Baggage"/>.
         /// </summary>
+        /// <remarks>
+        /// Note: <see cref="Current"/> returns a copy of the current Baggage.
```

Comment:
minor: I personally find it confusing when we use the word "copy" since it could mean many different things - shallow copy, deep copy, copy by ref, shadow copy, copy-on-write, etc. I wonder if the word "fork" would give more clarity here - e.g. "returns a forked version of the Baggage" and "any updates on the forked version will not affect the Current version". I don't feel too strong about it though.

Reply:
I tweaked it to use "fork" LMK what you think.

Reply:
Looks great, thanks!